ddtrace/contrib/wsgi/wsgi.py

```diff
@@ -127,7 +127,7 @@ def intercept_start_response(status, response_headers, exc_info=None):
                 if hasattr(result, "__class__"):
                     resp_class = getattr(getattr(result, "__class__"), "__name__", None)
                     if resp_class:
-                        resp_span.meta["result_class"] = resp_class
+                        resp_span._meta["result_class"] = resp_class
```

Comment:
Also indirection here:

```suggestion
                        resp_span.set_tag("result_class", resp_class)
```

Reply:
Could maybe do `_set_str_tag` instead which is a bit more established as an internal API.